### ISSUE TITLE
Error message when starting bot on Windows: response: { message: 'Missing Access', code: 50001 },   code: 403

### ISSUE THREAD

**Issue by KC080** [Androz2091/discord-music-bot#22]

**Comment by SlothsAreLazyTho** [comment]
Please do not include your bot token in your error next time.

**Comment by KC080** [comment]
> Please do not include your bot token in your error next time.

yeah sorry. fixed it. did not notice it when i was copying the text

**Comment by SabakuNoMaiku** [comment]
I have this error too. Any idea on how resolve it?

**Comment by D0wzy** [comment]
- You forgot the `application.commands` scope into your OAuth2 bot settings.
- You started the bot when it was not in the specified server into the `.env file` (`DISCORD_GUILD_ID`)
